### PR TITLE
New authors: Derek Masaki

### PR DESCRIPTION
**Pull request by DanielJPino-ux**
adding author to index

A one-line description of the changes you're making and how they'll affect users.

---

**Preview:**

### CODE REVIEW

**Comment by DanielJPino-ux** [comment]
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/master-3/authors/derek-mesaki/

File: content/authors/derek-mesaki/_index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+# View this page at https://digital.gov/authors/derek-mesaki
+# Learn how to edit our pages at https://workflow.digital.gov
+
+# slug — the specific user-id for an author.
+slug: derek-mesaki
```
Comment (saracope) [inline content/authors/derek-mesaki/_index.md:6]: the last name is misspelled I think

**Comment by ToniBonittoGSA** [comment]
Needs folder change/rename, too

**Comment by ToniBonittoGSA** [comment]
see https://github.com/GSA/digitalgov.gov/pull/2724

**Pull request closed without merging**